bundler/src/lib.rs

```diff
@@ -59,7 +59,7 @@ impl Bundler {
         .with_context(|| {
             format!("failed to put bytecode for actor {:?} into blockstore", actor_type)
         })?;
-        self.added.insert(actor_type, cid);
+        self.added.insert(actor_name(actor_type), cid);
```

Comment:
This is (a) not needed because synthetic CIDs are supported as `forced_cid`s, and (b) you're merging this to the v8 branch, which uses content-addressed CIDs.